types/ahocorasick/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for ahocorasick 1.0.2
+// Project: https://github.com/BrunoRB/ahocorasick
+
+declare class AhoCorasick {
+    constructor(keywords: string[]);
+
+    search(match: string): [number, string[]][]
```

Comment:
```suggestion
    search(match: string): Array<number, string[]>;
```

Reply:
@codershiba Thanks for fixing it! But the correct type is `Array<[number, string[]]>`.

Reply:
Yeah, missed that but seems to be fixed by the author now.